tutorial/getting-started/02-connect-frontends/react/readme.md

```diff
@@ -0,0 +1,8 @@
+# Getting Started with Drasi
+Follow the tutorial [instructions here](https://drasi.io/tutorials/connecting-frontends/react).
+
+### Steps for running in VS Code with a Dev Container
+1. Open this directory in Visual Studio Code
+2. Open the Command Palette by typing `Ctrl + Shift + P` (windows) or `Cmd + Shift + P` (mac)
```

Comment:
I opened this folder in VS Code but I wasn't able to start the dev container. Is this because there isn't a dev container in this directory?

Reply:
good catch, those instructions are wrong